### PR TITLE
Add Mollusk Walker info & update Contributing Instructions

### PR DESCRIPTION
**Pull request by RodEsp**
Info from game:
Mollusk
Mollusk-Walker-Wings-Small
Mollusk-Walker-Wings-Heavy
Mollusk-T2-Upgrades

### CODE REVIEW

**Comment by RodEsp** [comment]
Hey @kyleshrader, sorry about the commits still rolling into my PRs.
I already rebased ontop of the upstream/master but it's still doing it.
Hopefully following PRs will be better since I just did the rebase.

This is what it looks like on my local:
image

**Comment by RodEsp** [comment]
After this is merged I'll just hard-reset my fork's master branch to the `upstream/master` and it should be good.

File: README.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ The crafting tool can be found here: https://www.lastoasiscrafter.com
 
 # Contributing
 ## Adding/Updating Items
-Make a pull request with your additions or updates to src/itemObjGen.txt and leave a comment with where you got the data you are adding. Screenshots or links are helpful.
-
 I am using [ObjGen](https://beta5.objgen.com/json/) for now because it's an easier format to type up the object data in. Sort the resulting json by name, then category. Sorting the list saves me from doing it at runtime on every client's page load.
+
```
Comment (kyleshrader) [inline README.md:7]: Nice, thanks for adding this.

File: src/itemObjGen.txt [hunk 2]
```diff
@@ -2257,6 +2257,144 @@
...
+      name = Wood Shaft
+      count n = 18
+[]
+  name = Mollusk Walker Wings (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood
+      count n = 47
+      name = Fiber Weave
```
Comment (kyleshrader) [inline src/itemObjGen.txt:2296]: Missing an ingredient array declaration. Please fix this for all following ingredients as well

Comment (RodEsp) [reply src/itemObjGen.txt:2296]: Doy... I fix one typo and I make others... haha I'll fix it and add the rest of the Mollusk info later today, good catch!

**Comment by RodEsp** [comment]
@kyleshrader I don't really have a preference on working with JSON or in the ObjGen format, but you're right that ObjGen is a little easier/more efficient to edit by hand.

**Comment by RodEsp** [comment]
Alrighty, this should be ready to go @kyleshrader.

File: src/itemObjGen.txt [hunk 3]
```diff
@@ -2257,6 +2257,292 @@
...
+      name = Fiber
+      count n = 160
+    ingredients[]
+      name = Hide
+      count n = 24
+    ingredients[]
+      name = Wood
+      count n = 24
+[]
+  name = Mollusk Walker Upgrade - Gear - Tier 2
```
Comment (kyleshrader) [inline src/itemObjGen.txt:2371]: we've been going off the columns so this would have been hatch, but i dont know after the patch

**Pull request merged**